I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 42, in script
    jump in_progress
ScriptError: could not find label 'in_progress'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 42, in script
    jump in_progress
  File "C:\Users\newho\Desktop\Kim Nen\renpy-6.99.12.1-sdk\renpy\ast.py", line 1527, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\newho\Desktop\Kim Nen\renpy-6.99.12.1-sdk\renpy\script.py", line 858, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'in_progress'.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Kim Nen 1.0
